frontend/src/query_builder/VisualizationSettings.jsx

```diff
@@ -3,22 +3,11 @@ import React, { Component, PropTypes } from "react";
 import Icon from "metabase/components/Icon.jsx";
 import PopoverWithTrigger from "metabase/components/PopoverWithTrigger.jsx";
 
-import { hasLatitudeAndLongitudeColumns, isString } from "metabase/lib/schema_metadata";
+import visualizations from "../visualizations";
```

Comment:
i think it's nicer when we use explicit paths as much as possible, and it's especially nice for our core shared libraries.  so i think it's more appropriate for this to be `"metabase/visualizations"`